src/App.js

```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```

Comment:
**Changes Required**
Good effort with the function component, however;
- [ ] The `App Component` should be a `Class Component` as specified in the project requirement. Let us refactor it into a class component.